### DISCUSSION TITLE
Vouchers with Prefilled Names

### DISCUSSION THREAD

**Discussion by theschweitz** [pretix/pretix#2958] [Ideas]
This feature would better assist in our quest to ensure that Discounted Tickets are going to the actual applicants that filled out the application for the Discounted Tickets.

For instance: Right now someone can fill out a Discount Application with Person B's information, then receive a Discounted Voucher code, then put in their own name, and we would never know.

Vouchers could be generated in bulk, similar to how Imported Orders are created: by importing a CSV and selecting where the Voucher data comes from. We do have other ideas for how this application-based selection process could be made better — See #2956 .

**Comment by raphaelm** [comment]
We do want to bring some kind of "invitation" feature to pretix. What you are proposing is basically one of three designs we have in mind, next to "something like the order requiring approval, just the other way round" and "a mew concept entirely". We have not yet made up our mind which one is best for long term further development
